src/apps/vpn/tasks/purchase/taskpurchase.cpp

```diff
@@ -56,6 +57,7 @@ TaskPurchase::~TaskPurchase() { MZ_COUNT_DTOR(TaskPurchase); }
 
 void TaskPurchase::run() {
   NetworkRequest* request = new NetworkRequest(this, 200);
+  request->auth(MozillaVPN::authorizationHeader());
```

Comment:
Doe we need to add this to the functional test mocks as well?

Reply:
we do not have tests for the android/ios purchase...